bot.py

```diff
@@ -91,54 +98,18 @@ async def on_reaction_remove(reaction, user):
     pastas = json.load(f)
 with open("data/ualberta.ca.json", "r", encoding='utf-8') as f:
     catalog = json.load(f)
-with open("data/kattis.json", "r", encoding='utf-8') as f:
-    kattis_links = json.load(f)
-with open("data/problems.json", "r", encoding='utf-8') as f:
-    kattis_problems = json.load(f)
-with open("data/contests.json", "r", encoding='utf-8') as f:
-    kattis_contests = json.load(f)
-with open("data/specific.json", "r", encoding='utf-8') as f:
-    kattis_specific = json.load(f)
-
-##### ?snipe
-class DeletedMsg:
-    def __init__(self, msg, attachment_path=None):
-        self.msg = msg
-        self.attachment_path = attachment_path
-        self.time = datetime.utcnow()
-
-deleted_messages = {} # {channel id : DeletedMsg}
 
 @client.event
 async def on_message_delete(message):
-    attachment_path = None
-    if message.attachments:
-        attachment_path = await util.save_attachment(message.attachments[0])
-    deleted_messages[message.channel.id] = DeletedMsg(message, attachment_path)
```

Comment:
atttachment snipe removed?

Reply:
It was refactored into `cogs/snipe.py`. Tested and still works.